.gitignore

```diff
@@ -6,3 +6,4 @@
 *.swp
 .idea
 *.iml
+.DS_Store
```

Comment:
Again, different pull request. No matter how trivial, it does not belong to this feature.